src/WebJobs.Extensions.DurableTask/Listener/TaskOrchestrationShim.cs

```diff
@@ -165,4 +225,36 @@ public override void RaiseEvent(OrchestrationContext unused, string eventName, s
             this.context.RaiseEvent(eventName, serializedEventData);
         }
     }
+
+    internal class AsyncCommand
+    {
+        [JsonProperty("ActionType")]
```

Comment:
I worry the JavaScript purists might not be pleased with our Pascal casing here (I normally expect these kinds of JSON objects to be camel-cased). I'm guessing it would only be language worker authors that need to know this schema. Would it make sense to make it camel-case since that seems to be a more commonly accepted standard?